lib/provider/default_volume_provider_test.go

```diff
@@ -128,27 +128,13 @@ func TestGetSnapshot(t *testing.T) {
 	assert.Nil(t, getSnap)
 }
 
-func TestGetSnapshotWithVolumeID(t *testing.T) {
-	ccf := &DefaultVolumeProvider{sess: nil}
-
-	getSnapWithID, _ := ccf.GetSnapshotWithVolumeID("VolumeID", "SnapshotID")
-	assert.Nil(t, getSnapWithID)
-}
-
 func TestListSnapshots(t *testing.T) {
 	ccf := &DefaultVolumeProvider{sess: nil}
 
-	listSnapWithID, _ := ccf.ListSnapshots()
+	listSnapWithID, _ := ccf.ListSnapshots(50, "1", nil)
 	assert.Nil(t, listSnapWithID)
 }
```

Comment:
Validate values and "next"

Reply:
Checked implementation of other UTs. Looks like since it is only interface library, we have verified method signature here. Detailed response verification is done in volume-vpc code base